README.Rmd

```diff
@@ -13,1044 +12,68 @@ knitr::opts_chunk$set(
-
-A new version allows using more up to date
-capacity factors derived from IEA WEO numbers. This is required in order to
-calculate profits on actual quantities produced, not theoretical capacities.
-
-Output Columns:
-
-- scenario (transition scenario for which the capacity factors apply)
-- scenario_geography (geographic region for which the capacity factors apply)
-- technology (usually the PACTA sectors with production pathways)
-- year (temporal change of capacity factors)
-- capacity_factor (a number indicating the average power generation per unit of capacity)
-
-
-**Scenario data** that covers production road maps for all technologies that are
-to be analysed up until 2040. The scenario data need to cover those road maps
-for the scenarios selected in the parameter files and should follow the
-structure found in Scenarios_AnalysisInput_YYYY.csv. This data is used to
-extrapolate the production trajectories for companies beyond the PACTA time frame.
-
-**Price data** trajectories found in the file prices_data_YYYYQQ.csv, which
-contains projections of market prices per technology from the start data of the
-analysis until the year 2040. This is another required input to obtain profits.
-This will likely change in the future.
-
-**Net profit margins** are loaded via the model_parameters.yml file up until now.
-This may change in the future.
-
-**Excluded companies** in case a specific company-technology combination cannot
-be covered by the model either due to lack of data on the company side or because
-of other input data gaps, the combination of caompany_name and technology can be
-specified to be excluded from risk calculations. Such situations can be relevant
-for example when a company conducts business predominantly in one country that
-follows transition policies are laws that are not covered in the regionality of
-the business as usual and target scenarios. For instance, Germany is phasing out
-nuclear power in the first half of the 2020s. European and/or global scenarios do
-not capture this, although it can be seen in the relevant companies' capex plans.
-Such situations can lead to unintended behaviour of the model and as such it
-seems reasonable not to shock this part of the business, especially since we
-usually model shocks happening beyond this time frame.
-
-**Further notes on input data:**
-
-- It is of particular importance that the start year of the analysis is available
-in the PACTA results, in the scenario data and in the price data. If either data
-set starts after the start year of the analysis, the work flow will not work.
-- Equally, the scenarios selected for the analysis must be given in the PACTA
-results as well as in the scenario data and the price data.
 
-#### Steps
+### Look up valid ranges of model parameters
+You can look up allowed values of model parameters as such:
+```{r}
+# looking up allowed range for risk_free_rate
+risk_free_rate_range_lookup
```

Comment:
why not simply add all of them here? Maybe not best practice, but it would not really make the readme unbearably complicated

Reply:
DONE